apps/backoffice-v2/src/common/components/organisms/ImageViewer/ImageViewer.SelectedImage.tsx

```diff
@@ -28,7 +28,10 @@ export const SelectedImage = forwardRef<HTMLImageElement | HTMLIFrameElement, TS
       setIsError(false);
     }, [isLoading, selectedImage?.imageUrl]);
 
-    if (selectedImage?.fileType === 'pdf' || selectedImage?.imageUrl?.endsWith('.pdf')) {
+    if (
+      selectedImage?.fileType === 'application/pdf' ||
```

Comment:
use memtype as the "new" way to descover file type.
and keep old one for backword compatibility until we migrate all the relevant information of files.
add TODO so we wont forget to remove it later on.